packages/ui/src/helpers/shared/__tests__/validator.test.ts

```diff
@@ -0,0 +1,84 @@
+import { PasswordSettings } from '../../../types';
+import { ALLOWED_SPECIAL_CHARACTERS } from '../../../helpers/authenticator/constants';
+import { defaultPasswordValidator } from '../validator';
+
+const strictPasswordPolicy: PasswordSettings = {
+  passwordPolicyCharacters: [
+    'REQUIRES_LOWERCASE',
+    'REQUIRES_NUMBERS',
+    'REQUIRES_UPPERCASE',
+    'REQUIRES_SYMBOLS',
+  ],
+  passwordPolicyMinLength: 8,
+};
+
+const lenientPasswordPolicy: PasswordSettings = {
+  passwordPolicyCharacters: [],
+  passwordPolicyMinLength: 4,
+};
+
+describe('validateFormPassword', () => {
+  it.only('validates as expected with valid password and strict password policy', async () => {
```

Comment:
Remove `.only`?

Reply:
Oh gosh, thanks for the catch.

Reply:
Updated in [6078356](https://github.com/aws-amplify/amplify-ui/pull/2908/commits/6078356e6e1184668aa1eee66e12b43b8abaa2b8) 🙏